### DISCUSSION TITLE
OpenDTU flashen

### DISCUSSION THREAD

**Discussion by JFischer5383** [tbnobody/OpenDTU#2023] [Q&A]
Guten Tag,
ich möchte für meine opendtu 32 eine alte release-Version aufspielen, da die aktuelle Version nicht mit meinem script zur Nulleinspeisung funktioniert. 
Dazu hatte ich die bin datei herausgesucht und über die Oberflächen der DTU das update probiert. Leider ohne Erfolg - danach war die DTU nicht mehr erreichbar; das Standard-Passwort wird nicht mehr akzeptiert (ich hatte das Passwort nicht gewechselt)

Mit dem Flash-Programmen (Windows) komme ich leider nicht weiter, da hier die com-Ports einen Fehler anzeigen (auch nach Aktualisierung der Treiber). 

Besteht die Möglichkeit, die DTU über den Raspberry zu flashen?

LG
Jörg

**Comment by manfredogetup** [comment]
Hallo wir haben das High Speed Kabel Mac ca. 50cm lang genommen. Es war das einzige  das durchkontaktiert war und so auch die com Ports erreichbar waren. Alle anderen Kabel graben eine Fehlermeldung ab. Bedte Grüße Manfred Von meinem iPhone gesendetAm 31.05.2024 um 07:53 schrieb JFischer5383 ***@***.***>:﻿
Guten Tag,
ich möchte für meine opendtu 32 eine alte release-Version aufspielen, da die aktuelle Version nicht mit meinem script zur Nulleinspeisung funktioniert.
Dazu hatte ich die bin datei herausgesucht und über die Oberflächen der DTU das update probiert. Leider ohne Erfolg - danach war die DTU nicht mehr erreichbar; das Standard-Passwort wird nicht mehr akzeptiert (ich hatte das Passwort nicht gewechselt)
Mit dem Flash-Programmen (Windows) komme ich leider nicht weiter, da hier die com-Ports einen Fehler anzeigen (auch nach Aktualisierung der Treiber).
Besteht die Möglichkeit, die DTU über den Raspberry zu flashen?
LG
Jörg

**Comment by JFischer5383** [comment]
Vielen Dank für den Hinweis - ich werde mir ein neues Kabel besorgen.
Wenn ich den Gerätemanager ansehe, ist für den Port aber auch ohne Kabel ein gelbes Ausrufezeichen dran - ich habe dazu viel probiert, aber ich bekomme das nicht weg :-(

**Comment by JFischer5383** [comment]
gibt es in der Kommunity vielleicht einen Experten, der auf die Frage: "flash per raspberry" antworten kann?

**Comment by John-Corey** [comment]
Suche nach "esp32 flash raspi" liefert diverse Fundstellen mit einem Python-Script "esptool.py". Vielleicht bringt Dich das weiter.

Unter Windows flashe ich die Controller mit "flash_download_tool_3.9.6", Anleitungen dazu gibt es einige.
Das Kabel muss eines sein, was durchkontaktiert ist, da tut es jedes datenfähige USB-Kabel. Evt. hilft es, wenn Du im geräte-Manager bei dem Teil mit Ausrufezeichen per Rechtsklick die damit asoziierten Treiber deinstallierst.

Viel Erfolg!

**Comment by Al-Sol-We** [comment]
Hallo,

wo findet man die Flash-Daten für die Nulleinspeisung ?

Viele Grüße
